upup/models/cloudup/resources/addons/networking.cilium.io/k8s-1.12.yaml.template

```diff
@@ -127,6 +127,7 @@ data:
   enable-endpoint-routes: "true"
   auto-create-cilium-node-resource: "true"
   blacklist-conflicting-routes: "false"
+  masquerade: "false"
```

Comment:
Doesn't this conflict with the `masquerade` on line 120?

This seems a reasonable thing to do, but I don't see it in the upstream charts.

Reply:
It isn't directly in the charts, but it is in the documentation: https://docs.cilium.io/en/v1.6/gettingstarted/aws-eni/#prepare-deploy-cilium

I will change this to forcing the config on line 120 instead.